Automattic/jetpack
# Jetpack Search customization: show title on mobile

Fixes #20907.

#### Changes proposed in this Pull Request:
Make changes to the title in the new Customberg search customization experience:
* Always show the title including on mobile breakpoints (see #20907)
* Include a verb 'customize' in the title to indicate what the page is for
* Add a translator note to ensure the product title 'Jetpack Search' is not translated

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
On a site with Jetpack Instant Search enabled, visit `/wp-admin/admin.php?page=jetpack-search-configure`.

Check that the title looks like this across all breakpoints:

<img width="408" alt="Screen Shot 2021-09-02 at 16 43 58" src="https://user-images.githubusercontent.com/17325/131783524-80afe8e0-67a4-4ae4-8c04-e443569bd311.png">
